src/SFA.DAS.Assessors/InnerApi/Responses/GetStandardsListItem.cs

```diff
@@ -5,8 +5,11 @@ namespace SFA.DAS.Assessors.InnerApi.Responses
     public class GetStandardsListItem : StandardApiResponseBase
     {
         public string StandardUId { get; set; }
+        public string IfateReferenceNumber { get; set; }
         public int LarsCode { get; set; }
         public string Title { get; set; }
+        public decimal? Version { get; set; }
```

Comment:
As above, does this want to be a string?

Reply:
as above